curta/src/chip/memory/time.rs

```diff
@@ -9,7 +9,7 @@ use crate::math::prelude::*;
 pub type Time<F> = TimeStamp<ArithmeticExpression<F>>;
 
 #[derive(Debug, Clone, Copy, PartialEq, Eq, Hash, Serialize, Deserialize)]
-pub struct TimeStamp<T>(T);
+pub struct TimeStamp<T>(pub T);
```

Comment:
I actually think it's a little dangerous to make it pub, maybe we can add a method for access?

Reply:
Actually, would you be ok with changing it to `pub struct TimeStamp<T>(pub(crate) T);`?

Reply:
Yeah that'll be OK.